browser/src/control/Control.ChartContextToolbar.ts

```diff
@@ -0,0 +1,151 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * ChartContextToolbar - definition of chart context toolbar
+ * This toolbar should appear when a chart is selected
+ */
+
+class ChartContextToolbar extends JSDialogComponent {
```

Comment:
If we want to show UI attached to the "interactive element inside document" we should consider canvas sections first. They can be attached to the position in the document. JSDialogs are for UI which is on top of the screen but not attached to the document area. It's hard to keep it positioned correctly with all zoom/scaling/window resizing :)

As we discussed best example is table overlay in https://github.com/CollaboraOnline/online/blob/8c7acb9accffe66257f42158a665affdce6c6a5c/browser/src/layer/tile/TileLayer.TableOverlay.js#L369
and it's buttons https://github.com/CollaboraOnline/online/blob/master/browser/src/canvas/sections/TableInsertMarkerSection.ts